src/components/layout/nav/PrivateMenu.tsx

```diff
@@ -52,34 +53,41 @@ export default function PrivateMenu() {
   const { data: session, status } = useSession()
   const router = useRouter()
   const [anchorEl, setAnchorEl] = useState<Element | null>(null)
+  const { data: user } = useCurrentPerson()
 
   const handleMenu = (event: React.MouseEvent) => setAnchorEl(event.currentTarget)
   const handleClose = () => setAnchorEl(null)
+  const getAvatar = () => {
+    const person = user?.user
+    if (person?.picture) return person.picture as string
+  }
```

Comment:
At this point you can get rid of the function. Last thing here I promise :D

Reply:
You can provide the default directly where you use the person.picture like that:
`person.picture || {the default value for the avatar}`

Reply:
yes i was wondering if i should just do it this way but the function way seemed a little better since the default value 
for the avatar should be just undefined.

Reply:
> yes i was wondering if i should just do it this way but the function way seemed a little better since the default value for the avatar should be just undefined.

If the default value is undefined then you don't need to do anything else other than just `user?.user?.picture` which will fallback to undefined. 
Now when I see that you can also rename the data from `useCurrentPerson` to `session` so that you don't have a `user?.user`